_html/scss/modules/_my-profile-page.scss

```diff
@@ -0,0 +1,185 @@
+.profile-wrapper {
+    @include horizontal-center;
+    top: 12%;
+    width: 100%;
+    max-width: 785px;
+    min-height: 300px;
+    background-color: $white;
+    font-family: $font-main;
+    z-index: 1;
+
+    p {
+        color: $black;
+    }
+
+    @include tablet {
+        top: 38%;
+    }
+}
+
+.profile-wrapper--edit {
```

Comment:
You can nest this inside `.profile-wrapper` like: 
.profile-wrapper {
   //
  
   &--edit {
    //
  }
}